### PR TITLE
Update feature tests

### PR DESCRIPTION
**Pull request by jforgue**

### CODE REVIEW

File: app/views/passengers/index.haml [hunk 1]
```diff
@@ -6,14 +6,18 @@
...
   .passenger-filters
-    Permanent Only
-    = radio_button_tag :filter, 'Permanent', @filters.include?('Permanent')
-    Temporary Only
-    = radio_button_tag :filter, 'Temporary', @filters.include?('Temporary')
-    All
-    = radio_button_tag :filter, '', @filters.include?('All')
-    Show Inactive
-    = check_box_tag :show_inactive, true, !@filters.include?('Active')
+    %label{ for: 'filter_Permanent' }
```
Comment (dfaulken) [inline app/views/passengers/index.haml]: I don't think the radio button should be inside its own label.

File: spec/features/passenger_filters_spec.rb [hunk 2]
```diff
@@ -10,51 +10,51 @@
     create :passenger, :temporary, name: 'Gary Stue'
     create :passenger, :permanent, name: 'Mary Sue'
     visit passengers_url
-    choose('filter_Permanent')
-    click_on('Show Passengers')
-    expect(page).to have_selector('table#passengers tbody tr', count: 1)
-    expect(page).to have_content('Mary Sue')
-    expect(page).not_to have_content('Gary Stue')
+    choose 'Permanent Only'
```
Comment (dfaulken) [inline spec/features/passenger_filters_spec.rb]: Nice, didn't know that was the DSL for radio buttons.

File: app/views/passengers/index.haml [hunk 3]
```diff
@@ -3,16 +3,15 @@
...
 
 = form_tag passengers_path, method: :get do
-  %p
   = label_tag 'Filter by Type:'
   .passenger-filters
-    Permanent Only
+    %label{ for: 'filter_Permanent' } Permanent Only
     = radio_button_tag :filter, 'Permanent', @filters.include?('Permanent')
-    Temporary Only
+    %label{ for: 'filter_Temporary' } Temporary Only
```
Comment (mboneil10) [inline app/views/passengers/index.haml]: Why is `Temporary` capitalized?

**Comment by dfaulken** [comment]
CC issues are okay.

**Pull request merged**